pavelib/assets.py

```diff
@@ -172,14 +195,23 @@ def compile_templated_sass(systems, settings):
     """
     for sys in systems:
         sh(django_cmd(sys, settings, 'preprocess_assets'))
+        print("\t\tFinished preprocessing assets.")
 
 
-def process_xmodule_assets():
+def process_xmodule_assets(restart_django_server=False):
     """
     Process XModule static assets.
     """
     sh('xmodule_assets common/static/xmodule')
 
+    if restart_django_server:
+        # The python server needs to be reloaded (for some reason) in order to
```

Comment:
@feanil I can see that. However, I found that I always need to reload the Django server after running `xmodule_assets`, otherwise the CMS would give me a [`script error`](http://requirejs.org/docs/errors.html#scripterror) (if I'm recalling correctly).

Frankly, I didn't further research it. But I guessed that there's some kind of caching or preprocessing that causes this behavior as a side effect.